INSTALL

```diff
@@ -3,9 +3,10 @@
 Clone repository or unpack the tar ball and install with
 [pip](https://pip.pypa.io):
 
-    pip install propka-3.1/
+    pip install <source directory>
 
-This will install the `propka31` script in your executable
-directory. The `propka` Python module will also become available for
-import.
+This will install the `propka32` script in your executable
+directory.  The `propka` Python module will also become available for import
+and can be invoked in lieu of the `propka32` script via:
 
+    python -m propka
```

Comment:
The INSTALL file does not have an `.md` extension (is not rendered with markup).

It's also very similar -- but details are different -- from [README.md#installation](https://github.com/jensengroup/propka-3.1/blob/master/README.md#installation)

Reply:
I'll merge the two; thanks!

Reply:
Addressed in a2196cdc6fba738d83dc2f373bfde7a74b915dc9